services/ui_backend_service/data/db/tables/run.py

```diff
@@ -111,13 +111,12 @@ def select_columns(self):
             WHEN end_attempt_ok IS NOT NULL
             THEN end_attempt_ok.ts_epoch
             WHEN {table_name}.last_heartbeat_ts IS NOT NULL
-                AND @(extract(epoch from now())-{table_name}.last_heartbeat_ts)<={heartbeat_threshold}
+                AND @(extract(epoch from now())-{table_name}.last_heartbeat_ts)<={heartbeat_cutoff}
```

Comment:
changed this to `heartbeat_cutoff` as well so we don't end up in a situation where a run has a `finished_at` timestamp, yet is still marked as "running". At a glance it seemed that the query prior to changes in #338 was also relying on cutoff rather than threshold for this case. 

Does the change seem reasonable @romain-intel ?